fdbcli/ChangeFeedCommand.actor.cpp

```diff
@@ -146,7 +146,40 @@ ACTOR Future<bool> handlePopCommand(Database localDb, std::vector<StringRef> tok
 namespace fdb_cli {
 
 using CommandHandler = Future<bool> (*)(Database localDb, std::vector<StringRef> tokens, Future<Void> warn);
```

Comment:
Not quite sure if a std::function would be better than a raw function pointer, but anyway.